### DISCUSSION TITLE
Warranty worries

### DISCUSSION THREAD

**Discussion by luizpcarvalho** [jhassan8/crunchyroll-tizen#76] [General]
Hello! 
Do you know if installing this app on my Samsung TV could void the warranty?

**Comment by MatiasLGonzalez** [comment]
In theory, just enabling the option no, but if something gets broken by using some feature of the developer mode, then according to this post, yes. So I supposed that as long as this app doesn't break the TV, then your warrany should be intact :)
![imagen](https://github.com/jhassan8/crunchyroll-tizen/assets/48534497/67ec3e36-a00a-4530-bc33-8a6c0979f5f9)
https://us.community.samsung.com/t5/QLED-and-The-Frame/Can-t-use-developer-mode/m-p/2565612/highlight/true#M23158

**Reply by luizpcarvalho** [reply]
After installing this app do I need to keep developer mode active?

**Reply by MatiasLGonzalez** [reply]
@luizpcarvalho no, after the app has been installed there's no need to keep the developer mode on :)

**Reply by luizpcarvalho** [reply]
Thank you, very much!
